rhel8/profiles/cis.profile

```diff
@@ -516,7 +516,7 @@ selections:
     - wireless_disable_interfaces
 
     ## 3.6 Disable IPv6 (Not Scored)
-    - kernel_module_ipv6_option_disabled
+    - grub2_disable_ipv6
```

Comment:
Which way should be recommended?

Just because a particular method is in the CIS content does not make it correct or preferred (same for existing ComplianceAsCode rules, too!). 

Providing documentation around why an existing rule is no longer being suggested would be ideal. If that rule is no longer sufficient, should it also be deleted from the ComplianceAsCode catalog? Do other profiles rely on it that should also be updated to this new ``grub2_disable_ipv6`` rule?

Reply:
According to
https://access.redhat.com/solutions/8709
The method used in this rule is correct for rhel7 and rhel8.
The method used by rule kernel_module_ipv6_option_disabled is correct for older rhel systems. All the profiles where the kernel_module_ipv6_option_disabled rule is referenced belong to rhel6.